client.go

```diff
@@ -69,7 +69,7 @@ func (c *Client) BlockByHash(hash string, complete bool) (*rpcpb.BlockResponse,
 }
 
 // BlockByNumber .
-func (c *ClienClient) BlockByNumber(number int64, complete bool) (*rpcpb.BlockResponse, error) {
+func (c *Clien) BlockByNumber(number int64, complete bool) (*rpcpb.BlockResponse, error) {
```

Comment:
So you changed it to `Clien` instead of `Client`?

Reply:
i am wrong,it should be client